lib/vhdl-linter.ts

```diff
@@ -272,7 +273,14 @@ export class VhdlLinter {
         }
         if (pkgHeader) {
           pkg.correspondingPackage = pkgHeader;
+        } else {
+          this.addMessage({
+            range: pkg.range,
+            severity: DiagnosticSeverity.Warning,
+            message: `Can not find package for package body.`
+          })
```

Comment:
Can you provide an explicit test case for this?

Reply:
Added test case